src/i18n/locale-en.json

```diff
@@ -2936,6 +2936,8 @@
     "export.graphs.per.page": "Graphs per page:",
     "search.graphs.label": "Search Graphs",
     "export.repository.label": "Export repository",
+    "export.all.graphs.btn.label": "Download all graphs",
+    "export.graphs.too.many.warning": "The list is limited to 100,000 graphs to optimize the performance",
```

Comment:
Why is this hardcodded in the label and not a parameter?

Reply:
Refactored